clubs_guides/urls.py

```diff
@@ -1,3 +1,28 @@
 from django.conf.urls import url
 
-# Create you url mappings here.
+from clubs_guides.views import (
+    ClubsGuidesListView,
+    ClubsGuideView,
+    CategoryListView,
+)
+
+
+# Root URL: `/clubsguides` (this can be changed in the teach/urls.py file)
+# NOTE: The patterns below are tacked onto the root url mentioned above
+
+urlpatterns = [
+    # `/` - Gets a list of clubs guides
+    url("^$", ClubsGuidesListView.as_view(), name="clubsguide-list"),
+
+    # `/:pk` - Gets a single clubs guide by its primary key viz. `id`
+    url(r"^(?P<pk>[0-9]+)", ClubsGuideView.as_view(), name="clubsguide"),
```

Comment:
primary key. Maybe I should clarify more in the comment?
This is actually some "Django magic" that happens here since it recognizes pk as the primary key and automatically generates the queryset based on that without having you code the extra logic for getting the id from the params and using it in the queryset.